x/evm/keeper/state_transition.go

```diff
@@ -544,6 +548,9 @@ func (k *Keeper) ApplyMessageWithConfig(ctx sdk.Context,
 	}
 
 	gasUsed := sdk.MaxDec(minimumGasUsed, sdk.NewDec(int64(temporaryGasUsed))).TruncateInt().Uint64()
+	if ignoreMinGasMultiplier {
+		gasUsed = sdk.NewDec(int64(temporaryGasUsed)).TruncateInt().Uint64()
```

Comment:
nit but let's use a var for this val since it's used twice – also what does temporaryGasUsed actually mean?